src/redux/selectors/data/bookingsSelectors.js

```diff
@@ -100,6 +106,9 @@ export const selectUpComingBookings = createSelector(
   }
 )
 
+const finishedEventPredicate = (offer, booking) =>
```

Comment:
```suggestion
const isEventFinished = (offer, booking) =>
```
Ta méthode renvoie un booléen, la convention est d'exprimer cette intention dans le nom de la méthode